fuzzers/libfuzzer_libpng/Makefile.toml

```diff
@@ -26,6 +31,28 @@ wget https://deac-fra.dl.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1
 tar -xvf libpng-1.6.37.tar.xz
 '''
 
+# cmd doesn't support file download, "`" means continue to read next line
```

Comment:
I think we don't need these two downloading tasks for now (because we don't build it in the end)